server.go

```diff
@@ -46,6 +47,11 @@ func NewServer(c *structs.Config) (*Server, error) {
 	pc := controllers.NewPeopleController(session, c)
 	getRouter.HandleFunc("/people/{id:[0-9]+}/", pc.Get)
 
+	staticC := controllers.NewStaticController(c)
+	getRouter.HandleFunc("/about/", staticC.GetAbout)
+	getRouter.HandleFunc("/contact/", staticC.GetContact)
+	getRouter.HandleFunc("/getinvolved/", staticC.GetInvolved)
```

Comment:
Something I've been wondering (for the schedule side of things, mainly) is how to get these URL fragments into one place so we're not duplicating them all over the place (in menus in templates, in links on the schedule, etc)., without chronically overengineering the controllers.